x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -359,7 +359,6 @@ export function App({
     const docToSave = {
       ...getLastKnownDocWithoutPinnedFilters()!,
       description: saveProps.newDescription,
-      savedObjectId: saveProps.newCopyOnSave ? undefined : lastKnownDoc.savedObjectId,
```

Comment:
Removed `savedObjectId` from `docToSave` because it is not used and made tracking down how `attributeService.wrapAttributes` pulls out the savedObjectId extra tricky